docs/libraries/nodejs/examples.md

```diff
@@ -1,101 +1,149 @@
+* Needless to say, the seed should be set only for the first time. In order to open already initialized database, just a password is enough
 
-Add your custom password to the `.env` file.
+The storage is encrypted at rest and so you need a strong password and location where to put your storage. 
 
-## 1. Example: Create an Account
+> Please note: deal with the password with utmost care
 
-Run:
-```
-node 1-create-account.js
-```
+Technically speaking, the storage means a single file called `wallet.stronghold`. The generated seed (mnemonic) serves as a cryptographic key from which all accounts and related addresses are generated.
+
+One of the key principle behind the `stronghold`-based storage is that no one can get a seed from the storage. You deal with all accounts purely via `AccountManager` instance and all complexities are hidden under the hood and are dealt with in a secure way.
+
+In case one would like to store a seed also somewhere else, there is a method `AccountManager.generateMnemonic()` that generates random seed and it can be leveraged before the actual account initialization.
+
+Please note, it is highly recommended to store `stronghold` password and `stronghold` database on separate devices.
 
-Code:
+### Accounts
+The library uses a model of individual accounts to separate individual users/clients from each other. It is possible to generate multiple addresses for each account deterministically. 
+
+Once the backend storage is created, individual accounts for individual users can be created:
 ```javascript
-{{ #include ../../../bindings/nodejs/examples/1-create-account.js }}
+    let account = await manager.createAccount({
+        alias: 'Alice',  // an unique id from your existing user
+        clientOptions: { node: 'http://api.lb-0.testnet.chrysalis2.com', localPow: false }
+    })
 ```
+Each account is related to a specific IOTA network (mainnet / devnet) which is referenced by a node properties, such as node url (in this example, Chrysalis testnet balancer).
 
-## 2. Generate Address
-Run:
-```
-node 2-generate-address.js
-```
+For more information about `clientOptions`, please refer to [Wallet NodeJs API Reference](https://wallet-lib.docs.iota.org/libraries/nodejs/api_reference.html).
+
+`Alias` can be whatever fits to the given use case and should be unique. The `alias` is usually used to identify the given account later on. Each account is also represented by `index` which is incremented (by 1) every time new account is created. 
+Any account can be then referred to via `index`, `alias` or one of its generated `addresses`.
+
+Several api calls can be performed via `account` instance.
+
+> Note: it is a good practice to sync the given account with the Tangle every time you work with `account` instance to rely on the latest information available: `account.sync()`.
```

Comment:
note that it's a sync is performed automatically on `send`, `retry`, `reattach`, `promote` calls.

Reply:
Added a note https://github.com/iotaledger/wallet.rs/pull/437/commits/26da8466f6f36b82e71ab9daf081bfb74cb21403